README.md

```diff
@@ -9,11 +9,70 @@ Since the [cipd executable](https://chromium.googlesource.com/infra/luci/luci-go
 currently used to update V8 and its dependencies, and apply the z/OS port, after
 the initial [fetch v8](https://v8.dev/docs/source-code).
 
+I used depot_toolsport to fetch v8
+
 ```
 cd v8
 git pull
+```
+
+This gives:
+
+```
+remote: Finding sources: 100% (31/31)
+remote: Total 31 (delta 16), reused 29 (delta 16)
+Unpacking objects: 100% (31/31), 51.33 KiB | 320.00 KiB/s, done.
+From https://chromium.googlesource.com/v8/v8
+   da023d0d7c2..d178dc9c8ff  12.2-lkgr              -> origin/12.2-lkgr
+ * [new branch]              12.3.49                -> origin/12.3.49
+ * [new branch]              12.3.50                -> origin/12.3.50
+ + 87b461a80e2...d72213da29e canary                 -> origin/canary  (forced update)
+   da023d0d7c2..d178dc9c8ff  chromium/6261          -> origin/chromium/6261
+   7276abbe09f..0089b69a17f  lkgr                   -> origin/lkgr
+   098127338d8..0b8fdb94a15  main                   -> origin/main
+ + 28c4744132e...d72213da29e roll                   -> origin/roll  (forced update)
+   da023d0d7c2..d178dc9c8ff  refs/branch-heads/12.2 -> branch-heads/12.2
+ * [new tag]                 12.2.281.8             -> 12.2.281.8
+ * [new tag]                 12.2.281.8-pgo         -> 12.2.281.8-pgo
+ * [new tag]                 12.3.48                -> 12.3.48
+ * [new tag]                 12.3.48-pgo            -> 12.3.48-pgo
+ * [new tag]                 12.3.49                -> 12.3.49
+ * [new tag]                 12.3.49-pgo            -> 12.3.49-pgo
+ * [new tag]                 12.3.50                -> 12.3.50
+ * [new tag]                 12.3.50-pgo            -> 12.3.50-pgo
+You are not currently on a branch.
+Please specify which branch you want to merge with.
+See git-pull(1) for details.
+
+    git pull <remote> <branch>
+```
+
+
+
+
+
 pushd ..
 gclient sync
+
+```
+This doesn't make sense to me.
+We are in `$HOME/zopen/dev/v8`, the pushd .. will change dir to zopen/dev.
+Attempting `gclient sync` from `$HOME/zopen/dev/` dir and I get this result:
+
+(myenv) JD895801@USILCA31 dev
+$ gclient sync
+Error: client not configured; see 'gclient config'
```

Comment:
I think the problem is the lines in setenv.sh that remove gclient* [here](https://github.com/ZOSOpenTools/depot_toolsport/blob/main/setenv.sh#L20-L24). I suggest removing them. One won't want to be fetching v8 every time they do a new `git pull && gclient sync`. After removing those lines, do this only once:
```
cd $HOME/zopen/dev/
rm -rf v8
# The following 4 commands are the same as those at https://v8.dev/docs/source-code under step 4:
mkdir ~/v8
cd ~/v8
fetch v8
cd v8
```
Then, to apply the patches, you need to reset the head commit to that for which the patches (provided in https://github.com/ZOSOpenTools/v8port/pull/1) apply. So, while still inside v8 dir:
```
git reset fce38915e4c7c73c0e7f77bb13e2a78514968d35 --hard
```
Then follow the instructions in https://github.com/ZOSOpenTools/v8port/, starting with the `pushd ..` command.

I'll create a PR to add the above git reset to v8port's README.md.

Rather than PRs to discuss issues as here, I suggest opening a new issue by clicking on the New issue button at https://github.com/ZOSOpenTools/v8port/issues.